doc/user_manual/chSpecialElements.tex

```diff
@@ -2035,6 +2035,31 @@ \subsubsection{Additional Collimator Settings}
 JAW_FIT tcsg7 15 FIT_A FIT_B 1.0 1.0 on on
 \end{cverbatim}
 
+\paragraph{Crystal Collimators} \texttt{CRYSTAL name Bend XDim YDim Thick Tilt MisCut Orient}\\
+
+This flag enables the collimator specified in \texttt{name} to be treated as a crystal collimator.
+Crystal collimators require an additional set of parameters described in the following:
+
```

Comment:
can you add `\index{crystal collimator}` at the end of the first paragraph? It adds a reference in the index at the back of the manual.